### PR TITLE
Add a migration to have markets.yml migrated to the database

### PR DESCRIPTION
**Pull request by sramsden**
Add a migration to have config/markets.yml migrated to the database markets table rather than having to copy/edit yml then run seed task .. but leave the seed task just to ensure it doesn't fail if the seed yml does not exist.
The migration will simply do nothing if there is no config/markets.yml file available to migrate and it will not overwrite any existing markets in the database either (matching via market id)

### CODE REVIEW

**Comment by yivo** [comment]
@sramsden If this is useful I mind to keep some legacy stuff in master and other latest branches. Thanks for this feature. Could you please rebase? I will ask someone to review your code.

File: lib/peatio/version.rb [hunk 1]
```diff
@@ -1,3 +1,3 @@
 module Peatio
-  VERSION = '1.6.0'
+  VERSION = '1.7.0.alpha'
```
Comment (ysv) [inline lib/peatio/version.rb:2]: @sramsden we don't need version bump

**Comment by ysv** [comment]
@sramsden Please also remove https://github.com/rubykube/peatio/pull/1068/commits/081323383a943e9f2255ed9c63883ed1dee87b6b https://github.com/rubykube/peatio/pull/1068/commits/7778a88f2c09b3750fc9287e003f3fde941319f9

**Comment by ysv** [comment]
@yivo I thinks we need same pr for 1-7-stable, 1-8-stable and master, don't we ?

**Comment by sramsden** [comment]
@yssavchuk @yivo I thought I had created the original PR against your master and it certainly didn't include the version bump and it didn't include anyone else's commits either.
I'll clean it up and resubmit .. but which branch would you prefer I request it merged to?
My changes ..
https://github.com/rubykube/peatio/pull/1068/commits/7ad6e7906669543ee5a1f6ceee71bd874a40200f
.. simply add 2 new files (migration and spec) and changes seed rake task to check yml exists .. so it's a low risk change, but happy to PR it on your preferred branch.

**Comment by yivo** [comment]
@sramsden This happened because I changed base branch from master to 1-6-stable. This is OK that we see unexpected commits. Could you please rebase to 1.6 and push?

@yssavchuk We can keep this only in 1.6 branch. I don't want to support legacy things in newer branches.

**Pull request closed without merging**